python/pyspark/ml/feature.py

```diff
@@ -2385,7 +2385,31 @@ class RFormula(JavaEstimator, HasFeaturesCol, HasLabelCol):
     |0.0|0.0|  a|[0.0,1.0]|  0.0|
     +---+---+---+---------+-----+
     ...
-    >>> rf.fit(df, {rf.formula: "y ~ . - s"}).transform(df).show()
+    >>> model = rf.fit(df, {rf.formula: "y ~ . - s"})
```

Comment:
Do you mean use the `rf.fit(df)` other than this `rf.fit(df, {rf.formula: "y ~ . - s"})` ?